### PR TITLE
Export to stix bundle bug

### PR DESCRIPTION
**Pull request by yucohen**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/XSUP-14940)

## Description
The script failed when trying to export more than one indicator.  It failed because the indicators contain a field (score) that is not an original property of a stix indicator, and the Bundle method would fail.

### CODE REVIEW

File: Packs/CommonScripts/Scripts/StixCreator/StixCreator.py [hunk 1]
```diff
@@ -43,104 +42,94 @@
...
+    if length == 128:
+        return 'sha512'
+    return ''
 
+
+def main():
     user_args = demisto.args().get('indicators', 'Unknown')
     doubleBackslash = demisto.args().get('doubleBackslash', True)
     all_args = {}
-
```
Comment (yaakovi) [inline Packs/CommonScripts/Scripts/StixCreator/StixCreator.py:51]: Why did you remove all the spacing? Please restore.

**Review by yaakovi — APPROVED** [review]
add a test

**Comment by yucohen** [comment]
@ShirleyDenkberg Can someone please take a look at the RN? Thanks!

**Review by Ni-Knight — APPROVED** [review]

**Comment by ShirleyDenkberg** [comment]
@Ni-Knight @yaakovi Doc review completed.
